user_journey/user_journey.model.lkml

```diff
@@ -16,6 +16,15 @@ explore: funnel_analysis {
               OR (${days_of_use.submission_date} = DATE_SUB(current_date, INTERVAL 2 DAY)
                   AND DATE_ADD(${funnel_analysis.submission_date}, INTERVAL 56 DAY) >  DATE_SUB(current_date, INTERVAL 2 DAY)));;
   }
+  join: client_properties {
+    relationship: one_to_one
+    type: left_outer
+    sql_where: client_properties.submission_date BETWEEN DATE_SUB(DATE({% date_start funnel_analysis.date %}), INTERVAL {% parameter client_properties.diff_days %} DAY) AND DATE_SUB(DATE({% date_end funnel_analysis.date %}), INTERVAL {% parameter client_properties.diff_days %} DAY);;
+    sql_on: ${funnel_analysis.sample_id} = ${client_properties.sample_id}
+      AND ${funnel_analysis.client_id} = ${client_properties.client_id}
+      AND ${funnel_analysis.submission_date} = DATE_SUB(${client_properties.submission_date}, INTERVAL {% parameter client_properties.diff_days %} DAY);;
+    fields: [client_properties.fraction_is_default_browser, client_properties.count_is_default_browser, client_properties.is_default_browser, diff_days]
```

Comment:
I'm not sure we want to expose `client_properties.is_default_browser`. How do you see that being used in analysis?

Reply:
I don't think it is useful for analysis. However, removing it from this list results in a LookML Error: 

`Unknown field "client_properties.is_default_browser" referenced in "filters" parameter.
client_properties.view:13 user_journey:funnel_analysis`

Reply:
Okay, let's include it but hide it then (using `hidden: "yes"`).